<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-B2220QC" ignored="0" name="FirefoxTest" tests="4" failures="2" timestamp="2022-05-24T19:29:29 IST" time="0.225" errors="0">
  <testcase name="testC" time="0.028" classname="testClasses.TestClass2">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:86)
at org.junit.Assert.fail(Assert.java:95)
at testClasses.TestClass2.testC(TestClass2.java:39)
at java.util.ArrayList.forEach(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- testC -->
  <testcase name="testD" time="0.002" classname="testClasses.TestClass2"/>
  <testcase name="testA" time="0.03" classname="testClasses.TestClass">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testClasses.TestClass.testA(TestClass.java:46)
at java.util.ArrayList.forEach(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- testA -->
  <testcase name="testB" time="0.013" classname="testClasses.TestClass"/>
</testsuite> <!-- FirefoxTest -->
